Cartfile

```diff
@@ -1 +1 @@
-github "antitypical/Result" ~> 2.1.3
+github "antitypical/Result" "3.0.0"
```

Comment:
Can you please change this not to pin to a specific tag? `github "antitypical/Reault" ~> 3.0`